website/sampling-consensus.md

```diff
@@ -1,37 +1,79 @@
-# Atoma's Sampling Consensus
+##  Issue with Trust 
 
-In order to process requests, the Atoma Network protocol selects a given
-number of nodes uniformly at random. The selected nodes must execute the
-given requests, otherwise part of their collateral value is at risk
-of being slashed. Moreover, the selected nodes are guaranteed to have similar hardware and software specifications. The node sampling mechanism is part of our `Sampling Consensus` protocol. The latter allows the Atoma Network to satisfy the following features:
+In a decentralized network, you can’t just run an LLM and trust the output. If we ask the network to run inference using Llama2-70B, how can we be assured that it’s not actually using LLama2-13B? 
```

Comment:
Instead of decentralized, you can say in a trustless/permissionless/low reputation. Centralized services have sort of same problems

Reply:
Also you can't just run an LLM, doesn't sound formal enough. I suggest: no trust assumptions can be made on the correctness of the supposedly AI generated output.

Feel free to improve the above

Reply:
Reworded